CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.22)
+file(READ VERSION VERSION)
+string(STRIP ${VERSION} VERSION_STRIPPED)
```

Comment:
Using a tool like [bumpversion](https://pypi.org/project/bumpversion/) can keep this kind of special wiring logic to a minimum, but no need to hold up this PR on that. Someone can circle back to that in the future if it's interesting.

Reply:
Let's circle back to this on another PR